pmd-java/src/main/java/net/sourceforge/pmd/lang/java/metrics/api/JavaOperationMetricKey.java

```diff
@@ -93,6 +105,19 @@ public String name() {
             public boolean supports(ASTMethodOrConstructorDeclaration node) {
                 return metric.supports(node);
             }
+
+
+            @Override
+            public boolean equals(Object obj) {
+                return obj == this;
```

Comment:
Here's the same equals method. Please update it similar to the one in JavaClassMetricKey